### ISSUE TITLE
Check UDID in Web Client?

### ISSUE THREAD

**Issue by purirohan** [bitstadium/HockeyKit#131]
Hi!

Please correct me if I'm wrong, but my understanding is that Team Management currently works as follows:
1. User downloads a "generic" version of the app (it has no team restrictions or even a .team file)
2. The generic app checks the UDID, sends it up to the server, pulls down that user's name and team name, and serves them the appropriate updated app file
3. Now the user is on the right "track" for doing that team's work (testing, beta, etc.)

Basically, team management requires the user install the app before being put on the right team with the right app version.

Is there anyway for the check for UDID to happen on the web client? That is, a beta tester or a developer team member only sees and can install the app version intended for them at the start?

**Comment by DerAndereAndi** [comment]
Your assumptions are correct.

The only way to detect the UDID is implementing a process like HockeyApp does it which requires to create a profile on the server, sign it and install it on the device, then save the UDID in a session and allow the website domain to access it. Apple documentation for that process is available here: http://developer.apple.com/library/ios/#documentation/NetworkingInternet/Conceptual/iPhoneOTAConfiguration/Introduction/Introduction.html%23//apple_ref/doc/uid/TP40009505
